frontend/public/models/yaml-templates.ts

```diff
@@ -1243,6 +1243,60 @@ metadata:
 spec:
   hrefTemplate: 'https://example.com/logs?resourceName=\${resourceName}&containerName=\${containerName}&resourceNamespace=\${resourceNamespace}&podLabels=\${podLabels}'
   text: Example Logs
+`,
+  )
+  .setIn(
+    [referenceForModel(k8sModels.ConsoleQuickStartModel), 'default'],
+    `
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: example
+spec:
+  displayName: Deploy example application from image
+  tags:
+  - example
+  - image
+  - deployment
+  durationMinutes: 5
+  description: Example application.
+  prerequisites:
+  introduction: '### This Quick Start shows you how to deploy an existing image to OpenShift.'
+  tasks:
+    - title: Deploying an existing Image from an Image registry.
+      description: |-
+        ### To deploy the **hello-openshift** image:
+        1. Using the perspective switcher at the top of the navigation, go to **</> Developer**.
+        2. Go to the **+Add** page in the navigation.
+        3. Using the project drop-down list, select the project you would like to deploy the image in.
+        4. Click **Container Image** to deploy an image.
+        5. Type **openshift/hello-openshift** into the **Image name from external registry** field.
+        6. Scroll down and click **Create**.
+        The **Topology** view will load with your **hello-openshift** application. The application is represented by the light grey area with the white border. The deployment is a white circle.
+      review:
+        instructions: |-
+          #### To verify the image was successfully deployed:
+          1. Do you see a **hello-openshift** deployment?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      summary:
+        success: You have deployed example application using **openshift/hello-openshift** image!
+        failed: Try the steps again.
+    - title: Running the deployed application
+      description: |-
+        ### To run **hello-openshift** deployment application:
+        1. Click on the deployment's white circle in the **Topology** view.
+        2. Click on the top right quadrant of the **hello-openshift** deployment, which represents the route. It should open the URL and run the application.
+
+        The application will be run in a new tab.
+      review:
+        instructions: |-
+          #### To verify your application is running:
+          1. Make sure you are in the new tab.
+          2. Do you see **Hello OpenShift!** text in the new tab?
+      summary:
+        success: Your have run **hello-openshift** application!
```

Comment:
```suggestion
        success: Great work! You ran the **hello-openshift** application.
```

Reply:
I don't think we'll have congratulatory messages in quick starts anymore, but I don't know for sure.

Reply:
>  I don't think we'll have congratulatory messages in quick starts anymore, but I don't know for sure.

then how about using:
`success: You have deployed **hello-openshift** image.`

Reply:
Oh, I meant that I don't think any message will appear at all. But if a message does appear, it's fine to just use what we have ("Great work! You ran the hello-openshift application").